src/Build/BackEnd/BuildManager/BuildManager.cs

```diff
@@ -1664,6 +1664,9 @@ private void CheckSubmissionCompletenessAndRemove(BuildSubmission submission)
 
                     // Clear all cached SDKs for the submission
                     SdkResolverService.ClearCache(submission.SubmissionId);
+
+                    // Clear SDK resolver cache for the submission
```

Comment:
This line seems very similar to the previous one. A code comment on why this is would be nice.

Reply:
Good point, and after I took a second look at it, I decided this call should be moved.  I've also improved the comment